reana_commons/openapi_connections/reana_job_controller.json

```diff
@@ -0,0 +1,344 @@
+{
+  "definitions": {
+    "Job": {
+      "properties": {
+        "cmd": {
+          "type": "string"
+        },
+        "cvmfs_mounts": {
+          "items": {
+            "type": "string"
+          },
+          "type": "array"
+        },
+        "docker_img": {
+          "type": "string"
+        },
+        "experiment": {
+          "type": "string"
+        },
+        "job_id": {
+          "type": "string"
+        },
+        "max_restart_count": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "restart_count": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "status": {
+          "type": "string"
+        }
+      },
+      "required": [
+        "cmd",
+        "cvmfs_mounts",
+        "docker_img",
+        "experiment",
+        "job_id",
+        "max_restart_count",
+        "restart_count",
+        "status"
+      ],
+      "type": "object"
+    },
+    "JobRequest": {
+      "properties": {
+        "cmd": {
+          "default": "",
+          "type": "string"
+        },
+        "cvmfs_mounts": {
+          "default": [],
+          "items": {
+            "type": "string"
+          },
+          "type": "array"
+        },
+        "docker_img": {
+          "type": "string"
+        },
+        "env_vars": {
+          "default": {},
+          "type": "object"
+        },
+        "experiment": {
+          "type": "string"
+        },
+        "job_id": {
+          "format": "uuid",
+          "type": "string"
+        },
+        "job_name": {
+          "type": "string"
+        },
+        "job_type": {
+          "type": "string"
+        },
+        "prettified_cmd": {
+          "default": "",
+          "type": "string"
+        },
+        "shared_file_system": {
+          "default": true,
+          "type": "boolean"
+        }
+      },
+      "required": [
+        "docker_img",
+        "experiment",
+        "job_name"
+      ],
+      "type": "object"
+    }
+  },
+  "info": {
+    "description": "REANA Job Controller API",
+    "title": "reana-job-controller",
+    "version": "0.0.1"
```

Comment:
Now that this lives outside the specific repo (in this case out from `reanahub/reana-job-controller`) and we don't have the `git tag`'s, we should start tagging the API versions with this property using its [current version](https://github.com/reanahub/reana-job-controller/blob/master/reana_job_controller/version.py#L17).

Reply:
True